float32 score
string label

geometry_msgs/Pose2D center
float64 size_x
float64 size_y

geometry_msgs/PoseStamped point
float32 width